ngrinder-core/src/main/java/net/grinder/engine/agent/LocalScriptTestDriveService.java

```diff
@@ -237,11 +238,11 @@ private boolean isRunningOnWas() {
 		return ((URLClassLoader) LocalScriptTestDriveService.class.getClassLoader()).getURLs()[0].getProtocol().equals("jar");
 	}
 
-	private String runtimeClassPath() {
+	private String runtimeClassPath() throws UnsupportedEncodingException {
```

Comment:
Isn't it better to eat this exception silently?

Reply:
with try / catch internally.

Reply:
This exception seems will not occur when using `UTF-8`

when using try/catch internally how about like this?
```
try {
	runtimeClassPath.append(URLDecoder.decode(url.getFile(), "UTF-8")).append(File.pathSeparator);
} catch (UnsupportedEncodingException e) {
	throw new NGrinderRuntimeException("error while resolving runtime class path for script validation check", e);
}
```